### ISSUE TITLE
submission: allow direct upload of the single-file YAML format

### ISSUE THREAD

**Issue by eamonnmag** [HEPData/hepdata#31]
_From @GraemeWatt on December 18, 2015 17:14_

It would be nice to be able to directly upload the single-file YAML that we export from the old HepData, i.e. the hepdata-converter would need to be extended to convert the single-file YAML format to multi-file YAML format (HEPData/hepdata-converter#12) .  Since this is done anyway for the migration, could the code be integrated into the hepdata-converter?

_Copied from original issue: HEPData/hepdata#153_

**Comment by eamonnmag** [comment]
Since migration is so easy anyway, I would prefer to not have to implement this since it is of low priority. I've just implemented the single file oldhepdata upload since it would make it easier for existing tooling. But since no tools export the yaml yet, it seems unnecessary to ask for everything in a single file when we'll split it up anyway. We just increase the chance of something going wrong this way. By the way, I found this tool started by some people at FermiLab. We can develop on top of this for the ROOT conversion. https://github.com/lantone/HEPData
